Tests/Fixtures/TestBundle/EventSubscriber/MySubscriber.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Dunglas\ActionBundle\Tests\Fixtures\TestBundle\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * @author Ener-Getick <egetick@gmail.com>
```

Comment:
I always did it that way until now but yeah it's probably better to use my name instead.